internal/commands/commands.go

```diff
@@ -76,14 +76,14 @@ Realm app will be exported as well.`,
 			},
 			{
 				Use:         "create",
-				Display:     "app create",
+				Display:     app.CommandCreateDisplay,
 				Description: "Create a remote Realm app from your current working directory",
-				Help: `Creates a new Realm app in the cloud, and saves your configuration files on 
-a new disk. This command will create a new directory for your project. 
+				Help: `Creates a new Realm app in the cloud, and saves your configuration files on
+a new disk. This command will create a new directory for your project.
 To initialize a Realm app in your current working directory, use ‘app init’.
 
-You can specify a '--from' input to create a Realm app from 
-an existing app. If you do not specify a '--from' input, 
+You can specify a '--from' input to create a Realm app from
+an existing app. If you do not specify a '--from' input,
 the CLI will create a non-personalized Realm app.`,
```

Comment:
[Thought + Q] Should we not allow the users terminal determine the width/wrapping of Help <command> information at least for each block, instead of us following the golang character length?

Reply:
80 is a pretty standard width number, not just go-specific.  but by doing it on our end, we ensure a more consistent output, and in the case of terminal displays that don't soft-wrap the output, it absolutely helps there.

going to leave the approach as is for now (i do think the help text is in line for an overhaul if we ever get the bandwidth to do so -- sumedha had some thoughts about tweaking it from the out-of-the-box style `cobra` gives us)